docs/markdown/BUILD.bazel

```diff
@@ -99,9 +103,18 @@ nodejs_binary(
 nodejs_binary(
     name = "markdown",
     data = [
+        "//bazel/browsers/chromium:chromium-for-generation",
         "@npm//jsdom",
+        "@npm//mermaid",
+        "@npm//playwright-core",
     ],
     entry_point = "//docs/markdown:guides.mjs",
+    env = {
+        "CHROME_BIN": "$(CHROMIUM)",
+    },
+    toolchains = [
+        "//bazel/browsers/chromium:toolchain_alias",
```

Comment:
Are there ways to have sub-targets for guides with mermaid to only fetch the browsers when needed?

Reply:
I suppose we could have a flag on a macro so that we note if it uses mermaid, but I am not sure if thats worth doing, thoughts?

Reply:
If possible, would be nice. Browsers have issues the windows spaces, so that would be good, and it does require fetching the downloads

Reply:
Done.